content/faq/graph/index.adoc

```diff
@@ -0,0 +1,22 @@
+---
+title: "Graph"
+aliases:
+- /features/
+- /documentation/overview/
+date: 2018-06-20T19:04:38+02:00
+draft: false
+type: "known-issues"
+weight: 1
+layout: faq
+---
+
+:icons: font
+:imagesdir: /images/faq/
+
+This section describes some known issues in Kiali that may not be fixed in the very short term, and lead to limitations in a restricted number of corner cases. Hopefully, you wouldn't even notice them.
```

Comment:
restricted number of corner cases. Hopefully, you wouldn't even notice them. -->
restricted number of cases.  Hopefully, you won't even notice them.